src/.nuget/toolchain.ubuntu.14.04-arm.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
```

Comment:
Without this change I'm getting the following errors:

```
/home/corert/src/dirs.proj(36,5): error MSB3030: Could not copy the file "/home/corert/src/.nuget/toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.nuspec" because it was not found.
/home/corert/src/dirs.proj(36,5): error MSB3030: Could not copy the file "/home/corert/src/.nuget/toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development.nuspec" because it was not found.
```